core/utils.py

```diff
@@ -287,4 +279,19 @@ def normalizeKey(key: Union[None, 'db.KeyClass']) -> Union[None, 'db.KeyClass']:
     return db.Key(key.kind, key.id_or_name, parent=parent)
 
 
-from viur.core.skeleton import SkeletonInstance
```

Comment:
Please re-add the `SkeletonInstance` import.

It's used here:
https://github.com/viur-framework/viur-core/blob/c3d5dff409535a802d336f237410f2a0133d4a3f/core/utils.py#L180
and for type hints.

Reply:
aha, so why is it defined at the end of the file, and not inline? And even without any comment or hint.

Reply:
I've resolved this case inline in 2fa27d9

Reply:
Let's do it in another PR, separated from this one.